src/instructlab/data/generate.py

```diff
@@ -125,6 +125,15 @@
     "--model-family",
     help="Force model family to use when picking a generation template",
 )
+@click.option(
+    "--pipeline",
+    type=click.STRING,
+    default="simple",
```

Comment:
if not a click.choice then how about constants for each pipeline type in configuration.py :thinking: